github_pr.py

```diff
@@ -149,6 +149,40 @@ def _return_specific_comment_prs(all_prs, filters):
         return []
 
 
+def _check_approved_mergers_file(approved_users, comment_users):
+    if not isinstance(approved_users, list):
```

Comment:
if `approved_users` is not a list, what else can it be? If it is a string, I don't think https://github.com/dataxu/github-pr/pull/18/files#diff-c83be3194de0df69b64d430adffa6e96R156 will work correctly.